acceptance-tests/build.gradle

```diff
@@ -12,12 +12,11 @@
  */
 
 dependencies {
+  implementation project(':ethsigner:core')
```

Comment:
has someone worked out why this is so? Seems .. odd.

Reply:
It did not work with just testImplementation. I think this is the next thing to try, isn't it? I'm not sure why this is necessary. I'll try with testImplementation again ...

Reply:
It does work now with testImplementation! No idea what has changed ...
Fixed!